themes/solr/templates/operator/downloads.html

```diff
@@ -0,0 +1,160 @@
+      </ul>
+    </li>
+
+    <!-- TODO: Change log -->
+    <li><a href="{{ SITEURL }}/{{ SOLR_LATEST_RELEASE | replace(".", "_") }}/changes/Changes.html">Change log</a></li>
+  </ul>
+
+  <h3 id="solr-{{ SOLR_OPERATOR_PREVIOUS_MAJOR_RELEASE | replace(".", "") }}">Solr Operator {{ SOLR_OPERATOR_PREVIOUS_MAJOR_RELEASE }}
+    <a class="headerlink" href="#solr-{{ SOLR_OPERATOR_PREVIOUS_MAJOR_RELEASE | replace(".", "") }}" title="Permanent link">¶</a>
+  </h3>
+
+  <p>Solr Operator {{ SOLR_OPERATOR_PREVIOUS_MAJOR_RELEASE }} is the last release in the v0.{{ SOLR_OPERATOR_PREVIOUS_MAJOR_RELEASE.split(".")[1] }}.x series.</p>
+
+  <ul>
+    <li>Source release:
+      <a href="https://github.com/apache/solr-operator/archive/v0.2.8.tar.gz">solr-operator-{{ SOLR_OPERATOR_PREVIOUS_MAJOR_RELEASE }}.tgz</a>
+      <!-- TODO: Replace bullet point with following when v0.2.x is no longer relevant
+      <a href="https://www.apache.org/dyn/closer.lua/solr/operator/{{ SOLR_OPERATOR_PREVIOUS_MAJOR_RELEASE }}/solr-operator-{{ SOLR_OPERATOR_PREVIOUS_MAJOR_RELEASE }}.tgz">solr-{{ SOLR_OPERATOR_PREVIOUS_MAJOR_RELEASE }}.tgz</a>
+      [<a href="https://downloads.apache.org/solr/operator/{{ SOLR_OPERATOR_PREVIOUS_MAJOR_RELEASE }}/solr-operator-{{ SOLR_OPERATOR_PREVIOUS_MAJOR_RELEASE }}.tgz.asc">PGP</a>]
+      [<a href="https://downloads.apache.org/solr/operator/{{ SOLR_OPERATOR_PREVIOUS_MAJOR_RELEASE }}/solr-operator-{{ SOLR_OPERATOR_PREVIOUS_MAJOR_RELEASE }}.tgz.sha512">SHA512</a>]
+      -->
+    </li>
+
+    <!-- TODO: Change to Apache when v0.2.x is no longer relevant -->
+    <li>Docker releases:
+      <a href="https://hub.docker.com/r/bloomberg/solr-operator">bloomberg/solr-operator</a>
+    </li>
+
+    <li>Helm Chart release:
+      <a href="https://artifacthub.io/packages/helm/apache-solr/solr-operator/{{ SOLR_OPERATOR_PREVIOUS_MAJOR_RELEASE  | replace("v", "")  }}">Artifact Hub</a>
+    </li>
+
+    <li>CRDs:
+      <ul>
+        <li><a href="{{ SITEURL }}/operator/downloads/crds/{{ SOLR_OPERATOR_PREVIOUS_MAJOR_RELEASE }}/all.yaml">All</a></li>
+        <li><a href="{{ SITEURL }}/operator/downloads/crds/{{ SOLR_OPERATOR_PREVIOUS_MAJOR_RELEASE }}/solrclouds.yaml">Solr Cloud</a></li>
+        <li><a href="{{ SITEURL }}/operator/downloads/crds/{{ SOLR_OPERATOR_PREVIOUS_MAJOR_RELEASE }}/solrprometheusexporters.yaml">Solr Prometheus Exporter</a></li>
+        <li><a href="{{ SITEURL }}/operator/downloads/crds/{{ SOLR_OPERATOR_PREVIOUS_MAJOR_RELEASE }}/solrbackups.yaml">Solr Backup</a></li>
+      </ul>
+    </li>
+  </ul>
+
+  <h2 id="past-versions">Past versions
+    <a class="headerlink" href="#past-versions" title="Permanent link">&para;</a>
+  </h2>
+
+  <p>Archives for all past versions of Solr are available at <a href="https://archive.apache.org/dist/solr/operator">the Apache archives</a>. See the <a href="https://github.com/apache/solr-operator/releases">Github releases for pre-v0.3.0 releases</a></p>
```

Comment:
"past version of Solr-Operator"...
Is it even relevant to mention releases from pre-0.3 here?

Reply:
Not sure, just thought I'd include it in case people get confused.